scripts/start-deployBukkitSpigot

```diff
@@ -28,7 +28,19 @@ function buildSpigotFromSource {
   mkdir ${tempDir}
   cd ${tempDir}
 
-  jvmOpts="-Xms${INIT_MEMORY:-$MEMORY} -Xmx${MAX_MEMORY:-$MEMORY}"
+  jvmOpts=""
+
+  if isPercentage "${INIT_MEMORY:-$MEMORY}"; then
+    jvmOpts+="-XX:InitialRAMPercentage=$(getPercentageValue "${INIT_MEMORY:-$MEMORY}") "
```

Comment:
I just happened to be in these docs a few minutes ago. These need to ultimately go in JVM_XX_OPTS. 

https://docker-minecraft-server.readthedocs.io/en/latest/configuration/jvm-options/#:~:text=If%20needing%20to%20set%20%2DXX%20options%2C%20then%20pass%20those%20with%20the%20JVM_XX_OPTS%20variable%20instead

I haven't looked yet to see where this local var ends up.